script.js

```diff
@@ -9,4 +9,86 @@ hamburger.addEventListener('click', () => {
 document.querySelectorAll('.navLink').forEach(n => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   navmenu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+
+const projects = [
+  {
+    title: 'Multi-Post Services',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-up required. This has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    imgSrc: './images/Img Placeholder (1).png',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'Ecommerce Web',
+    description: 'This is an ecommerce website',
+    imgSrc: './images/images2.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'Fin. xchange blog',
+    description: 'This is for currency trading',
+    imgSrc: './images/image5.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'Charzy Global Services',
+    description: 'My graphics and printing business site',
+    imgSrc: './images/image3.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'St. Barnabas Catholic Chaplaincy',
+    description: 'A Catholic church website in North Cyprus',
+    imgSrc: './images/image3.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'Charzy Portfolio',
+    description: 'This is Charzy\'s portfolio',
+    imgSrc: './images/images4.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  },
+  {
+    title: 'Blog',
+    description: 'This blog allows post to be shared',
+    imgSrc: './images/image5.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  }
+];
```

Comment:
- Please add a description paragraph on each card, when they get clicked they should appear the same as shown in the figma design, this is a simple task for you as you added it on the first item in the array as the only thing you can do  is to add the same key in the object called description and you can add some descriptions. with the same length as shown in the figma design.